src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Use the `name()` method instead of `toString()` to get the string representation of the enum constants, as `toString()` can be overridden and may not return the enum constant's name as expected.